docs/maintainers/maintainers-guide.md

```diff
@@ -70,39 +70,39 @@ There are two ways to add contributors to a Contributor Insight Page:
 1. Create a new Contributor Insight Page with selected contributors.
 2. Add selected contributors to an existing Contributor Insight Page.
```

Comment:
I'm thinking to move line 68-71 to before the image so it won't be missed. Thoughts?

Reply:
That would help @adiati98. It'll make the instructions clearer for contributors.